import "session_event.proto";
import "serverinfo_replay_match.proto";

message Event_ReplayAdded {
    extend SessionEvent {
        optional Event_ReplayAdded ext = 1100;
    }
    optional ServerInfo_ReplayMatch match_info = 1;
}
